src/packagedcode/models.py

```diff
@@ -117,14 +117,6 @@ class AssertedLicense(Model):
     notice = StringType(default=None)
     url = StringType(default=None)
 
-    def as_dict(self):
```

Comment:
What is this replaced by? the equivalent schematics function?
If this is the case, as_dict is rather clear as a name, I would not want to have the schematics specific to percolat in the codebase everywhere. Instead we should have an as_dict function that wraps schematics calls.